director/user_views.py

```diff
@@ -38,8 +38,4 @@ def get(self, request: HttpRequest, *args, **kwargs):
                 'filter_options': FILTER_OPTIONS
             })
         else:
-            url = reverse('user_signin')
-            token = self.request.GET.get('token')
-            if token:
-                url += '?token={}'.format(token)
-            return redirect(url)
+            return redirect(reverse('open_main'))
```

Comment:
URL names can be passed straight to `redirect` without needing to call `reverse` first:

`return redirect('open_main')`

Reply:
Thanks, done